proposals/utils/pdf_diff_sections.py

```diff
@@ -570,8 +580,14 @@ def __init__(self, obj, sub_title, proposal):
         self.proposal = proposal
 
     def make_row_for_field(self, field):
+
+        if field == "kind":
+            return KindRow(self.obj, self.obj.attachment, field)
+        else:
+            obj = self.obj.attachment
+
         if field in self.get_row_fields():
```

Comment:
Not to wake sleeping dogs here, but why do we check if the requested field is in `self.get_row_fields()` here? Is this method called anywhere else than from `self.make_rows()`, which just loops over those exact fields?